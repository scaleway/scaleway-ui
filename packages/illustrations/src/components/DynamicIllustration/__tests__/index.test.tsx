import { describe, it } from '@jest/globals'
import { DynamicIllustration } from '..'
import {
  shouldMatchEmotionSnapshotDark,
  shouldMatchEmotionSnapshotLight,
} from '../../../../.jest/helpers'

describe('DynamicIllustration', () => {
  it('should work with light theme', () =>
    shouldMatchEmotionSnapshotLight(<DynamicIllustration name="empty" />))

  it('should work with dark theme', () =>
    shouldMatchEmotionSnapshotDark(<DynamicIllustration name="empty" />))
})
