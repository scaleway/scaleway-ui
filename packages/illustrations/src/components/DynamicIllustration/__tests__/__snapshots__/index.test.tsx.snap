// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DynamicIllustration should work with dark theme 1`] = `
<DocumentFragment>
  <img
    alt="empty"
    src="test-file-stub"
  />
</DocumentFragment>
`;

exports[`DynamicIllustration should work with light theme 1`] = `
<DocumentFragment>
  <img
    alt="empty"
    src="test-file-stub"
  />
</DocumentFragment>
`;
